lib/benchee/benchmark/runner.ex

```diff
@@ -155,17 +160,20 @@ defmodule Benchee.Benchmark.Runner do
   defp iteration_time(scenario, scenario_context = %ScenarioContext{
                                   num_iterations: num_iterations
                                 }) do
-    measure_iteration(scenario, scenario_context) / num_iterations
+    {microseconds, memory_usage} = measure_iteration(scenario, scenario_context)
+    {microseconds / num_iterations, memory_usage}
```

Comment:
shouldn't we also divid memory_usage by num_iterations?

Reply:
Yep - good catch!